src/pages/PostDetails/PostDetails.jsx

```diff
@@ -0,0 +1,35 @@
+import { LikeOutlined } from '@ant-design/icons'
+import { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { Loading } from '../../components'
+import { IconText } from '../PostsList/components'
```

Comment:
Fix imports. You import `IconText` from `PostsList` so, you can move `IconText` to global `components` folder